### PR TITLE
Docs - Include examples in the table of content (and add examples)

### PR DESCRIPTION
**Pull request by alexandrehassan**
Being able to quickly navigate to a certain example or send a link for an example is really useful. 
- Adds the examples to a command's table of content (similar to [gnmic docs](https://gnmic.kmrd.dev/cmd/capabilities/#single-host))
- Split examples and a couple for certain commands (includes PR #1029)
- All commands typed into terminal start with `❯ ` to distinguish them from the output (following the format on the `exec` page)

### CODE REVIEW

File: docs/cmd/completion.md [hunk 1]
```diff
@@ -43,10 +43,10 @@ Start a new shell for this setup to take effect.
...
 #### Fish completions
 ```bash
-containerlab completion fish | source
+❯ containerlab completion fish | source
 ```
 
 To load completions for each session, execute once:
 ```
-containerlab completion fish > ~/.config/fish/completions/containerlab.fish
+❯ containerlab completion fish > ~/.config/fish/completions/containerlab.fish
```
Comment (hellt) [inline docs/cmd/completion.md:51]: the rule I started to follow is do not use shell sign if the command doesn't have output displayed. This is in accordance to some linter rules, as well as it makes it possible to copy the command using the copy icon and not using your mouse to copy the portion of the command without the shell sign

so please revert this change

File: docs/cmd/deploy.md [hunk 2]
```diff
@@ -67,10 +67,28 @@ Example command-line usage: `CLAB_VERSION_CHECK=disable containerlab deploy`
...
+
+#### Deploy a lab without specifying topology file
+
+Given that a single topology file is present in the current directory.
+
+```bash
+❯ containerlab deploy
+```
+
+#### Deploy a lab using shortcut names
```
Comment (hellt) [inline docs/cmd/deploy.md:90]: using short flag names

File: docs/cmd/destroy.md [hunk 3]
```diff
@@ -37,14 +37,35 @@ Destroy command provided with `--all | -a` flag will perform the deletion of all
...
-```
\ No newline at end of file
+#### Destroy all labs on the container host
+
+```bash
+❯ containerlab destroy -a
+```
+
+
+#### Destroy a lab using shortcut names
```
Comment (hellt) [inline docs/cmd/destroy.md:67]: using short flag names

File: docs/cmd/exec.md [hunk 4]
```diff
@@ -43,9 +45,23 @@ INFO[0000] clab-srl02-srl2: stdout:
 6: dummy-mgmt0: <BROADCAST,NOARP> mtu 1500 qdisc noop state DOWN group default qlen 1000
     inet 172.20.20.2/24 brd 172.20.20.255 scope global dummy-mgmt0
        valid_lft forever preferred_lft forever
+```
+#### Execute a command on one node by name
```
Comment (hellt) [inline docs/cmd/exec.md:49]: Execute a command on a node referenced by its name

File: docs/cmd/exec.md [hunk 5]
```diff
@@ -43,9 +45,23 @@ INFO[0000] clab-srl02-srl2: stdout:
...
+```bash
+❯ containerlab exec -t srl02.yml --label clab-node-name\=srl2 --cmd 'ip -4 a show dummy-mgmt0'
+INFO[0000] Parsing & checking topology file: srl02.yml  
+INFO[0000] Executed command 'ip -4 a show dummy-mgmt0' on clab-srl02-srl2. stdout:
+6: dummy-mgmt0: <BROADCAST,NOARP> mtu 1500 qdisc noop state DOWN group default qlen 1000
+    inet 172.20.20.5/24 brd 172.20.20.255 scope global dummy-mgmt0
+       valid_lft forever preferred_lft forever 
+```
 
+#### Execute a SR Linux CLI Command
```
Comment (hellt) [inline docs/cmd/exec.md:62]: let's not be specific about the node 
just `Execute a CLI command`

File: docs/cmd/exec.md [hunk 6]
```diff
@@ -75,9 +91,11 @@ Last Booted       : 2021-06-24T10:25:26.904Z
 Total Memory      : 24052875 kB
 Free Memory       : 21911914 kB
 ----------------------------------------------------
+```
 
+#### Execute a SR Linux CLI Command with json formatted output
```
Comment (hellt) [inline docs/cmd/exec.md:96]: Execute a command with json formatted output

File: docs/cmd/inspect.md [hunk 7]
```diff
@@ -45,19 +46,27 @@ containerlab inspect --all
...
+```bash
+❯ containerlab inspect --name srlceos01
 +---+---------------------+--------------+---------+------+-------+---------+----------------+----------------------+
 | # |        Name         | Container ID |  Image  | Kind | Group |  State  |  IPv4 Address  |     IPv6 Address     |
 +---+---------------------+--------------+---------+------+-------+---------+----------------+----------------------+
 | 1 | clab-srlceos01-ceos | 90bebb1e2c5f | ceos    | ceos |       | running | 172.20.20.4/24 | 2001:172:20:20::4/80 |
 | 2 | clab-srlceos01-srl  | 82e9aa3c7e6b | srlinux | srl  |       | running | 172.20.20.3/24 | 2001:172:20:20::3/80 |
 +---+---------------------+--------------+---------+------+-------+---------+----------------+----------------------+
 
+```
```
Comment (hellt) [inline docs/cmd/inspect.md:64]: here lets add an example where `inspect` command takes in a path to a topology file

**Review by hellt — CHANGES_REQUESTED** [review]

File: docs/cmd/inspect.md (hunk 7 above)
Comment (alexandrehassan) [reply docs/cmd/inspect.md]: Example has been added we now have `Provide information about a specific running lab by its name` and `Provide information about a specific running lab by its topology file`

**Review by hellt — APPROVED** [review]
